installation.md

```diff
@@ -35,7 +35,9 @@ First, download the Laravel installer using Composer:
 
     composer global require "laravel/installer=~1.1"
 
-Make sure to place the `~/.composer/vendor/bin` directory in your PATH so the `laravel` executable can be located by your system.
+Secondly, make sure to place the `~/.composer/vendor/bin` directory in your PATH so the `laravel` executable can be located by your system.
+
+For Windows  users make sure to add `%USERPROFILE%\AppData\Roaming\Composer\vendor\bin` to your `path` Environment variable.
```

Comment:
shouldn't be 2 spaces after Windows, also, now you need a comma after the word "users"